### PR TITLE
Created AccountManager Model

### PR DESCRIPTION
**Pull request by kingcrowz**

### CODE REVIEW

File: NJInsurancePlatform/Models/AccountManager.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        [Required(ErrorMessage = "Missing - Gender")]
+        public string? Gender { get; set; }
+
+        [DisplayName("Created Date")]
+        [Required(ErrorMessage = "Missing - Created Date")]
+        public DateTime CreatedDate { get; set; }
+
+        [DisplayName("List of Customer MUIDs")]
+        [Required(ErrorMessage = "Missing - List of Customer MUIDs")]
+        public string? ListOfCustomerMUIDs { get; set; }
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/Models/AccountManager.cs:32]: I thought this would be a list of CustomerMUIDS?

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]

File: NJInsurancePlatform/Models/AccountManager.cs (hunk 1 above)
Comment (Marcus-Johnson) [reply NJInsurancePlatform/Models/AccountManager.cs:32]: Strings or IEnumerable..

Comment (kingcrowz) [reply NJInsurancePlatform/Models/AccountManager.cs]: Changed the type to IEnumerable and added the imports at the top of the file

File: NJInsurancePlatform/Models/AccountManager.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        [Required(ErrorMessage = "Missing - Gender")]
+        public string? Gender { get; set; }
+
+        [DisplayName("Created Date")]
+        [Required(ErrorMessage = "Missing - Created Date")]
+        public DateTime CreatedDate { get; set; }
+
+        [DisplayName("List of Customer MUIDs")]
+        [Required(ErrorMessage = "Missing - List of Customer MUIDs")]
+        public List<string>? CustomerMUIDs { get; set; }
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/Models/AccountManager.cs:34]: public IEnumerable<CustomerMUID>? ListOfCustomerMUIDs { get; set; }

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]

**Comment by Marcus-Johnson** [comment]
`public IEnumerable<CustomerMUID>? ListOfCustomerMUIDs{ get; set; }`

**Review by Marcus-Johnson — APPROVED** [review]

**Pull request merged**